mqtt/mqtt-bridge/src/persist/loader.rs

```diff
@@ -78,9 +77,34 @@ where
 
             // refresh next batch
             // if error, either someone forged the database or we have a database schema change
-            let next_batch = self.next_batch()?;
+            let mut new_batch = self.next_batch()?;
+
+            // drop those loaded keys which do not exist in the new batch
```

Comment:
what are loaded keys?

Reply:
loaded keys a list of offsets for publications that are loaded from storage but not yet been send and removed from the storage